transforms/language/html2parquet/kfp_ray/Makefile

```diff
@@ -2,10 +2,20 @@ REPOROOT=${CURDIR}/../../../../
 WORKFLOW_VENV_ACTIVATE=${REPOROOT}/transforms/venv/bin/activate
 include $(REPOROOT)/transforms/.make.workflows
 
+
 # Include the common configuration for this transform
-include ../transform.config
+#include ../transform.config
+
+SRC_DIR=${CURDIR}/../
+# Use the docker image that is built for ray runtime
+TRANSFORM_RUNTIME=ray
+## override settings in .make.default as they assume old structure with ray being the current folder
+DOCKER_IMAGE_NAME=$(TRANSFORM_NAME)-$(TRANSFORM_RUNTIME)
```

Comment:
TRANSFORM_NAME does not seem to be set.  It was originally in ../transform.config.

Reply:
@daw3rd this is not received as an argument for the make -C:

$(MAKE) -C kfp_ray TRANSFORM_NAME=$(TRANSFORM_NAME) workflow-test;